packages/demos-babylon/src/gesturesDemo.js

```diff
@@ -26,13 +27,12 @@ async function createScene() {
   // the other pre-built host characters. Available character IDs are:
   // "Cristine", "Fiona", "Grace", "Maya", "Jay", "Luke", "Preston", "Wes"
   const characterId = 'Maya';
-  const characterConfig = HOST.HostObject.getCharacterConfig('./assets/character-assets', characterId);
+  const characterConfig = HOST.HostObject.getCharacterConfig('./character-assets', characterId);
```

Comment:
as an aside, I wonder if we should change `HOST` to something more specific, This is a global when using our webpack build and it isn't outside the realm of possibility that it will conflict with something else the customers has on the page

Reply:
Agree, but I don't want to take that on for this PR. I've created an internal ticket to track this need: P61381675

Reply:
yep sounds good. this is a breaking change for consumers so we should do this before our first release